mtp_bank_admin/settings/base.py

```diff
@@ -284,6 +284,15 @@
 CLOUD_PLATFORM_MIGRATION_MODE = os.environ.get('CLOUD_PLATFORM_MIGRATION_MODE', '')
 CLOUD_PLATFORM_MIGRATION_URL = os.environ.get('CLOUD_PLATFORM_MIGRATION_URL', '')
 
+# general ledger account code for prisoner monies holding bank account
+PRISONER_MONEY_HOLDING_ACCOUNT = '1841102059'
+
+# global setting to turn on all November 2 HMPPS policy changes
+NOVEMBER_SECOND_CHANGES_LIVE = os.environ.get('NOVEMBER_SECOND_CHANGES_LIVE', '').lower() in ('1', 'true')
```

Comment:
The flag is set slightly differently in the branches Toby and I worked on, but again short lived code so think it's OK!

Reply:
it's the same as the one in transaction-uploader: it's compatible and more flexible cos almost all flags used to expect 'True' as the value, but some new ones to expect '1' instead. this one works with both! i want to do a mini overhaul so that it's consistent later… i'm imagining something like:
```
SOME_FLAG = env_flag('SOME_FLAG', default=True)  # which could then work with 0/1 and False/True
```

Reply:
Cool, that sounds like a good plan 👍